syntax = "proto3";

message DeleteVMSnapshotRequest {

}
